src/etc/inc/config.console.inc

```diff
@@ -532,8 +532,7 @@ function vlan_setup() {
 		$vlan['if'] = chop(fgets($fp));
 
 		if ($vlan['if']) {
-			if (!array_key_exists($vlan['if'], $iflist) or
-			    !is_jumbo_capable($vlan['if'])) {
+			if (!array_key_exists($vlan['if'], $iflist) || !is_jumbo_capable($vlan['if'])) {
```

Comment:
C'mon, is that really needed? :) Not least look how many lines in the code (right close to it, let alone elsewhere and throughoput all the codebase) are > 80 chars. Does anyone in this day and age really need lines kept to 80 for reasons that still apply in this day and age of large screen resolutions (which almost all devs will have)?

That said, if it really needs to be kept to 80, then yes, for sure. But to my mind I feel that it is then harder to read, not easier, if lines are artificially split midway and most of the screen width is unusable.

Let me know if you really want it!